sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -258,3 +258,22 @@ def test_sample_weight():
         voting='soft')
     msg = ('Underlying estimator \'knn\' does not support sample weights.')
     assert_raise_message(ValueError, msg, eclf3.fit, X, y, sample_weight)
+
+
+def test_fit():
+    """Test sample weights inputs as list and array"""
```

Comment:
Could you convert docstring into a comment. We prefer this as docstring messes up clarity in verbose mode of `nosetests`....